.github/CODEOWNERS

```diff
@@ -152,6 +152,13 @@
 # ServiceLabel: %area-Authorization
 # ServiceOwners:                       @vurhanau
 
+# PRLabel: %area-Deploy
```

Comment:
please create the `area-Deploy` github issue label in both azure-mcp and azure-mcp-pr repos

Reply:
I notice area-Deploy label has been created and it seems I don't have permission to create label as I cannot find the create button.